### ISSUE TITLE
[WinUI3|x64 msix] Crash on run

### ISSUE THREAD

**Issue by Slals** [CommunityToolkit/Lottie-Windows#539]
Hi,

Does this package support x64? For the same build for `x64` and `x86` I have a crash with `x64` but not with `x86`, due to Lottie-Windows, here is the error:

```
System.BadImageFormatException
Could not load file or assembly 'CommunityToolkit.WinUI.Lottie, Version=7.1.0.0, Culture=neutral, PublicKeyToken=4aff67a105548ee2'.
```

**Comment by cuprumator** [comment]
Would be great to have x64 support. I have the same BadImageFormatException on x64 project configuration.

**Comment by aborziak-ms** [comment]
Dupe of #515. This should be fixed with release of 8.0
